src/display/canvas.js

```diff
@@ -913,6 +913,10 @@ function composeSMask(ctx, smask, layerCtx, layerBox) {
 }
 
 function getImageSmoothingEnabled(transform, interpolate) {
+  // In section 8.9.5.3 of the PDF spec, it's mentioned that the interpolate
+  // flag should be used when the image is upscaled.
+  // In Firefox, smoothing is always used when downscaling images.
```

Comment:
I'm probably missing something here, but in this function or in its callers I can't find code that would indicate that this is Firefox-only as it seems to be called in all browsers. Am I missing something here, or should we remove the "In Firefox" part here?

Reply:
Does my comment here: https://github.com/mozilla/pdf.js/issues/16273#issuecomment-2031885881, answer your question?

Reply:
Yes, it does; thank you for providing that analysis!

@calixteman Would it be an idea to change the comment here to `In Firefox, smoothing is always used when downscaling images (bug 1360415).`? That way we have a reference to this browser behavior in the code for future reference.

(It might also be good to add "Fixes #16273." to the commit message/PR description so we also have a reference to the original issue since I wasn't able to find it until it was linked here.)